sinonjs/sinon
# Finish migration to CircleCI and Codecov

This PR improves the Circle CI config to make better use of workflows, to run the integration tests and to drop node 6. It also removes the use of Travis and Coveralls (which has been failing with 500s lately).

 #### How to verify - mandatory
1. Check out this branch
1. Observe that the linter passes
1. Observe that tests pass in node 8, 10, 12
1. Observe that integration tests pass
1. Observe that codecov reports on the coverage

#### Checklist for author
